core/new-gui/src/app/dashboard/service/user-dictionary/user-dictionary.service.ts

```diff
@@ -1,12 +1,56 @@
 import { Injectable } from '@angular/core';
-import { Response, Http } from '@angular/http';
+import { HttpClient } from '@angular/common/http';
 
 import { Observable } from 'rxjs/Observable';
+import { Subject } from 'rxjs/Subject';
 import { UserDictionary } from '../../type/user-dictionary';
 
+const apiUrl = 'http://localhost:8080/api';
+
+const uploadDictionaryUrl = apiUrl + '/upload/dictionary';
```

Comment:
Rather than defining the API URL here, just used the existing method in `AppSettings` class in the `common` folder

Reply:
Thanks. The `AppSettings` is used in new commit